amykarnaze/bon-appetit
# Feature/th/panty can report if has ingredients

### What's this PR do?
Creates a function in Pantry that when given a list of ingredients in the format that they are created in the user data file checks the list against the stock of the panty and returns a boolean if both the items and amounts needed are met in the pantry stock

### Where should the reviewer start?
Changes made to the Pantry testing suite and the pantry.js

### How should this be manually tested?
The 3 test in pantry test check the functionality of the method and can be run with npm test

resolves #26
